pkg/errcode/errcode.go

```diff
@@ -13,6 +13,11 @@ import (
 // ErrCode defines the type to represent error codes
 type ErrCode int
 
+// Error code groups
+const (
+	Egress string = "egress"
+)
+
```

Comment:
Seems unnecessary, just the error code should suffice to retrieve the error. The group doesn't serve much.

Reply:
We weren't sure if we should include `errcode_group` or not. Our original thinking was that it would help group errors when creating dashboards, and provide more specificity when error codes are used across groups (`ErrMarshallingXDSResource` is used in LDS and the Injector for example).

Reply:
To avoid additional complexity, at this moment I think it makes sense to not include `errcode_group` in the errcode metric. This label can be added later if necessary.